lib/render/vp2RenderDelegate/proxyRenderDelegate.cpp

```diff
@@ -537,10 +537,14 @@ bool ProxyRenderDelegate::getInstancedSelectionPath(
     // therefore drawInstID is usdInstID plus 1 considering VP2 defines the
     // instance ID of the first instance as 1.
     const int drawInstID = intersection.instanceID();
+    const int usdInstID = drawInstID - 1;
+#if defined(USD_IMAGING_API_VERSION) && USD_IMAGING_API_VERSION >= 13
```

Comment:
Is there any user-visible change using the new code?

Reply:
@HdC-adsk: No, there should be no change in what gets drawn in the viewport. This was just cleaning up the API around querying scene delegates for an instancer's prim path.

There's more detail on the core USD change that goes with this here:
https://github.com/PixarAnimationStudios/USD/commit/82e343c42b730a3674a766aab6be1b4f1c0ca9ef